assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -35,19 +33,88 @@ const StyledPane = styled(Pane)`
   overflow: hidden;
 `;
 
-const Inner = styled.div`
+const Container = styled.div`
+  overflow: scroll;
   height: 100%;
-  display: grid;
-  grid:
-    'header   ' auto
-    'infinitescroll' 1fr
-    / 1fr;
+  padding: 0 1.5em 0 1.5em;
+  margin-top: 1em;
 `;
 
+/**
+ * Pane that contains the media 3P integrations.
+ *
+ * @param {Object} props Component props
+ * @return {*} The media pane element for 3P integrations.
+ */
 function Media3pPane(props) {
+  // TODO(#1698): Ensure scrollbars auto-disappear in MacOS.
+  // State and callback ref necessary to recalculate the padding of the list
+  //  given the scrollbar width.
+  const [scrollbarWidth, setScrollbarWidth] = useState(0);
+  const refContainer = useRef();
+  const refCallbackContainer = (element) => {
+    refContainer.current = element;
+    if (!element) {
+      return;
+    }
+    setScrollbarWidth(element.offsetWidth - element.clientWidth);
+  };
+
+  // TODO(#2368): get resources from useMedia
```

Comment:
[nit]: useMedia -> useMedia3p that will be added in https://github.com/google/web-stories-wp/issues/2611.